cli/tests/unit/signal_test.ts

```diff
@@ -21,88 +21,145 @@ unitTest(
         Deno.signal(1);
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.alarm(); // for SIGALRM
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.child(); // for SIGCHLD
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.hungup(); // for SIGHUP
       },
       Error,
-      "not implemented",
-    );
-    assertThrows(
-      () => {
-        Deno.signals.interrupt(); // for SIGINT
-      },
-      Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.io(); // for SIGIO
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.pipe(); // for SIGPIPE
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.quit(); // for SIGQUIT
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.terminate(); // for SIGTERM
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.userDefined1(); // for SIGUSR1
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.userDefined2(); // for SIGURS2
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
     assertThrows(
       () => {
         Deno.signals.windowChange(); // for SIGWINCH
       },
       Error,
-      "not implemented",
+      "Windows only supports ctrl-c(SIGINT) and ctrl-break(SIGBREAK)!",
     );
   },
 );
 
+unitTest(
+  { ignore: Deno.build.os !== "windows", perms: { run: true, net: true } },
+  async function signalWindowsStreamTest(): Promise<void> {
+    const resolvable = deferred();
+    // This prevents the program from exiting.
+    const t = setInterval(() => {}, 1000);
+
+    let c = 0;
+    const sig = Deno.signal(Deno.Signal.SIGINT);
+    setTimeout(async () => {
+      await defer(20);
+      for (const _ of Array(3)) {
+        // Sends SIGUSR1 3 times.
```

Comment:
SIGUSR1?

Reply:
Oh, hh.  In fact, this is a copy of signalStreamTest, this is my negligence